coalahtml/_coalahtml/index.html

```diff
@@ -9,9 +9,9 @@
     <link rel='stylesheet' type='text/css'
```

Comment:
If you move this, gh-pages demo wont work anymore.
Can we discuss this approach ? It doesnt seem to be a good one to me